pkg/planner/core/indexmerge_intersection_test.go

```diff
@@ -43,6 +43,40 @@ func TestPlanCacheForIntersectionIndexMerge(t *testing.T) {
 	require.True(t, tk.HasPlanForLastExecution("IndexMerge"))
 }
 
+func TestIndexMergeWithOrderProperty(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t (a int, b int, c int, d int, e int, key a(a), key b(b), key c(c), key ac(a, c), key bc(b, c), key ae(a, e), key be(b, e)," +
+		" key abd(a, b, d), key cd(c, d))")
+	tk.MustExec("create table t2 (a int, b int, c int, key a(a), key b(b), key ac(a, c))")
+
+	tk.MustExec("")
+
+	var (
+		input  []string
+		output []struct {
+			SQL    string
+			Plan   []string
+			Result []string
```

Comment:
We didn't insert any data. So I think we can remove the `Result` field to make the test result cleaner.